web-console/future-openshift/contextual-help/contextual-help.md

```diff
@@ -0,0 +1,85 @@
+# Contextual Help
+The OpenShift web console has a couple of unique ways of giving users help, in addition to standard patterns like Field Level Help. The purpose of this document is to provide information about the Hint Pattern and the Infopop pattern, as well as guidance as to when it's appropriate to use each pattern.
```

Comment:
I do think it would be worth referencing Andy's status popover hint since I could see someone looking for that guidance here. Maybe just a sentence at the end of this intro with something like...If you are looking to provide additional help on a status field (ex: migrating), see <status popover help design here>